cookiecutter.json

```diff
@@ -19,5 +19,6 @@
         "Apache Software License 2.0",
         "GNU General Public License v3",
         "Not open source"
-    ]
+    ],
+    "use_frontend": "y"
```

Comment:
suggestion: make the default "n" ?

Reply:
So I did this in bce3424
That causes the build to fail because if we just generate the package accepting all the defaults, we can't run the js build.
We can work round that by using a replay file in CI, which I've done in a44381b
That allows us to give cookiecutter a set of answers to use, but it also gives another thing we have to maintain/keep in sync as we modify `cookiecutter.json`, which will be annoying.

Now that I have seen the implementation, this is one of those things where no as a default might be marginally nicer for the user but I'd actually rather roll back those two commits and stick with yes as the default. Unless you are really strongly of the opinion this is worth the tradeoff.

Reply:
I missed this yesterday. It is a fair point and I agree we need a balance between maintainability and user convenience. Revert and roll 👍🏼